File: Backjoon1978/Backjoon1978.c
#include <stdio.h>

int main(){
    int N = 4;
    int n[4] = {1, 3, 5, 7};
    int count = N;

    for(int i = 0; i < N; i++){
        if(n[i] <= 1)
            count--;
        for(int ii = 2; ii < n[i]; ii++)
            if (n[i] % ii == 0){
                count--;
                break;
            }
    }
    printf("%d ", count);
}

File: Baekjoon_step/Backjoon1978/Backjoon1978.py
N = 4
n = [1, 3, 5, 7]
count = N

for i in range(0, N):
    if(n[i] <= 1):
        count -= 1
    for ii in range(2, n[i]):
        if (n[i] % ii == 0):
            count -= 1
            break
print(count)